src/tiny-inflate@1.0.3.esm.js

```diff
@@ -0,0 +1,397 @@
+/*
```

Comment:
Why are we adding a static dependency on someone else's code? If we're adding WOFF2 decompression (which has already been decided should be handled by using an external library in concert with Opentype.js to reduce bloat for people who don't need the decompression) we should use NPM, not statically including a library into our code.

Reply:
Need some clarification on this.

Reply:
Sorry I missed that one.
The reason is that [tiny-inflate](https://github.com/foliojs/tiny-inflate/) API does not provide an ESModule (`import`/`export`) version.

Lucky for us there are ESM-compatible versions (see: [tiny-inflate-es](https://github.com/code4fukui/tiny-inflate-es)) but sadly it is not published on NPM

Reply:
What is it used for though?

Reply:
It is only used for the WOFF (1) table decompression
See this paragraph : https://www.w3.org/TR/WOFF/#DataTables

WOFF2 use a much better and much complex algorythm.

Reply:
I don't really like the static dependency either, but I see why you needed to do it.
We could take a look at https://github.com/101arrowz/fflate - it claims to be *way* faster than tiny-inflate, while being about the same size when only using inflation via tree shaking.

Reply:
Just tried it out, it's as easy as
`import { decompressSync as inflate } from 'fflate';`

Reply:
Let's handle that separately and go with the static dependency for now.